sunpy/visualization/animator/base.py

```diff
@@ -434,10 +456,13 @@ def __init__(self, data, image_axes=[-2, -1], axis_ranges=None, **kwargs):
         for i in self.slider_axes:
             self.frame_slice[i] = 0
 
+        slider_functions = kwargs.pop("slider_functions", [])
+        slider_ranges = kwargs.pop("slider_ranges", [])
         base_kwargs = {
-            'slider_functions': [self.update_plot] * self.num_sliders,
-            'slider_ranges': [[0, dim] for dim in np.array(data.shape)[self.slider_axes]]
+            'slider_functions': ([self.update_plot] * self.num_sliders) + slider_functions,
```

Comment:
So does this mean the slider functions specified by the user in the ```slider_ranges``` kwargs are only extra slider functions, e.g. for a colourbar?  Data axes' sliders are still derived internally?

Same for ```slider_ranges```

Reply:
Yes, indeed it does.